gradle/integration-test.gradle

```diff
@@ -12,10 +12,7 @@ task integrationTest(type: Test) {
   group = 'verification'
 }
 
-['**/ScanIT_Gradle_Versions_3_3*.class',
- '**/ScanIT_Gradle_Versions_4_1*.class',
- '**/ScanIT_Gradle_Versions_4_4*.class',
- '**/ScanIT_Gradle_Versions_5_0*.class',
```

Comment:
These versions of gradle aren't compatible with Java 11.

Reply:
> These versions of gradle aren't compatible with Java 11. Plan is to add new/supported gradle versions for integration testing in a separate branch/pr (as I expect issues while adding them)

This means we would also need to update the `Compatibility` section in the README file

Reply:
> Plan is to add new/supported gradle versions for integration testing in a separate branch/pr (as I expect issues while adding them)

While at it, we might also want/need to update the Gradle version we're using to build the plugin.

Reply:
Just bumped up the project version as this is a breaking change:
d023b801052630f3d12bb8abc8fb32d068775d40

Reply:
> > Plan is to add new/supported gradle versions for integration testing in a separate branch/pr (as I expect issues while adding them)
> 
> While at it, we might also want/need to update the Gradle version we're using to build the plugin.

Agreed we probably need to update it, but I'd prefer to do that in another PR. The current version works with JDK 11.

Reply:
> This means we would also need to update the `Compatibility` section in the [README file](https://github.com/sonatype-nexus-community/scan-gradle-plugin?tab=readme-ov-file#compatibility):

👍 d970155d
 
> It probably be useful to mention the latest version where older Gradle versions where supported and the version where Gradle 5 is the lowest supported version with Java 11.

👍 795e49f7